clippy_lints/src/methods/mod.rs

```diff
@@ -1693,6 +1710,13 @@ impl<'tcx> LateLintPass<'tcx> for Methods {
             }
         }
     }
+
+    fn enter_lint_attrs(&mut self, cx: &LateContext<'tcx>, attrs: &'tcx [Attribute]) {
+        get_inner_attr(cx.sess(), attrs, "msrv", |msrv| {
+            check_msrv(cx.sess(), &msrv);
+            self.msrv = Some(msrv)
+        });
+    }
```

Comment:
I would put this function as-is in a macro in utils, so in the `LintPass` implementations, we only have to write 

```rust
msrv_attrs_check!() // or a better name
```

and not copy paste the whole function.